save-preprocessor/src/main/kotlin/com/saveourtool/save/preprocessor/controllers/DownloadProjectController.kt

```diff
@@ -158,6 +160,20 @@ class DownloadProjectController(
                 .subscribe()
         }
 
+    /**
+     * @param testFilesRequest
+     * @return [PublicTestDto] filled with test files
+     */
+    @PostMapping("/getTest")
```

Comment:
I suggest to create a new controller here, that will be used to access downloaded resources. `DownloadProjectController` already has several responsibilities.

Reply:
Will implement it